Mastodon/Scene/Onboarding/ServerRules/Cell/ServerRulesTableViewCell.swift

```diff
@@ -10,7 +10,7 @@ import MastodonAsset
 import MastodonLocalization
 
 final class ServerRulesTableViewCell: UITableViewCell {
-    
+    static let reuseIdentifier = "ServerRulesTableViewCell"
```

Comment:
I'm a bigger fan of
```suggestion
    static let reuseIdentifier = String(describing: ServerRulesTableViewCell.self)
```

but feel free to leave as-is.